docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml

```
@@ -529,11 +534,16 @@ if: |
   trial_court.classification.strip() != "Class 1"
 id: new owner facts for non-class 1 counties
 question: |
+  % if person_answering == "tenant":
   Did your landlord ever sell the house while you were a tenant at this address?
+  % else:
+  Did the tenant's landlord ever sell the house while they were a tenant at this address?
+  % endif
+  
 fields:
-  - Was your house sold to a new landlord while you lived in it?: previous_landlord_sold_property
+  - ${"Was your house sold to a new landlord while you lived in it?" if person_answering == "tenant" else "Was the tenant's house sold to a new landlord while they lived in it?"}: previous_landlord_sold_property
```

```suggestion
  - label: |
      % if if person_answering == "tenant":
      Was your house sold to a new landlord while you lived in it?
      % else:
      Was the tenant's house sold to a new landlord while they lived in it?
      % endif
    field: previous_landlord_sold_property
```

Ternary/one-line if statements aren't so great for translation. We should avoid those.